src/raftstore/store/fsm/apply.rs

```diff
@@ -61,11 +61,11 @@ const SHRINK_PENDING_CMD_QUEUE_CAP: usize = 64;
 pub struct PendingCmd {
     pub index: u64,
     pub term: u64,
-    pub cb: Option<Callback>,
+    pub cb: Option<Callback<RocksEngine>>,
```

Comment:
Why the `Callback` depends on `RocksEngine`?

Reply:
because it contains ReadCallback, which contains ReadRepsonse, which contains RegionSnapshot, which contains an impl of Snapshot.